3rdParty/Radon/Radon/source/Section.cpp

```diff
@@ -17,19 +17,21 @@ namespace radon
 	{
 	}
 
+
 	Key *Section::getKey(const std::string & name)
 	{
-		for (auto & key : keys)
+		for (int i = 0; i < (int)keys.size(); i++)
 		{
-			if (key.getName() == name)
-				return &key;
+			if (keys[i].getName() == name)
+				return &keys[i];
 		}
 
-		return nullptr;
+		return NULL;
 	}
 
+
 	void Section::addKey(Key key)
 	{
 		keys.push_back(key);
 	}
-}
+}
```

Comment:
Are these really intentional? Do we have it setup like this in the project?

Reply:
Radon is not part of DevilutionX code. It's a vendored third-party library.

Reply:
Are all files under the `3rd-party` folder excluded from editorconfig and compiler warnings on our side?

Reply:
They should be excluded from `.clang-format` and `.editor-config` (they're not currently -- this is an oversight).

However, we still want to have the warnings so that we can fix them upstream.